make fails when lib and lib64 directories are both present

After this PR:
https://github.com/Bears-R-Us/arkouda/pull/3423

`make` fails if both `lib` and `lib64` are present and the necessary dependencies are in `lib`.  

@stress-tess @brandon-neth @bmcdonald3

If you're curious, here is the error I was seeing:
```
                            init :   0.001 seconds
/usr/bin/ld: cannot find -lhdf5: No such file or directory
/usr/bin/ld: cannot find -lhdf5_hl: No such file or directory
/usr/bin/ld: cannot find -liconv: No such file or directory
/usr/bin/ld: cannot find -lparquet: No such file or directory
/usr/bin/ld: cannot find -lhdf5: No such file or directory
/usr/bin/ld: cannot find -lhdf5_hl: No such file or directory
clang++: error: linker command failed with exit code 1 (use -v to see invocation)
error: Make Binary - Linking


```

--- a/Makefile
+++ b/Makefile
@@ -57,10 +57,9 @@ define add-path
 ifneq ("$(wildcard $(1)/lib64)","")
   INCLUDE_FLAGS += -I$(1)/include -L$(1)/lib64
   CHPL_FLAGS    += -I$(1)/include -L$(1)/lib64 --ldflags="-Wl,-rpath,$(1)/lib64"
-else
-  INCLUDE_FLAGS += -I$(1)/include -L$(1)/lib
-  CHPL_FLAGS    += -I$(1)/include -L$(1)/lib --ldflags="-Wl,-rpath,$(1)/lib"
 endif
+INCLUDE_FLAGS += -I$(1)/include -L$(1)/lib
+CHPL_FLAGS    += -I$(1)/include -L$(1)/lib --ldflags="-Wl,-rpath,$(1)/lib"
 endef
 # Usage: $(eval $(call add-path,/home/user/anaconda3/envs/arkouda))
 #                               ^ no space after comma

